apps/web/src/app/typo.css

```
@@ -0,0 +1,80 @@
+/* Typo(Text Styles) */
+
+.large-title-1 {
+  @apply text-2xl font-bold;
+}
+
+.large-title-1A {
+  @apply leading-A text-2xl font-bold;
+}
+
+.title-1 {
+  @apply text-xl font-bold;
+}
+.title-2 {
+  @apply text-xl font-medium;
+}
+.title-3 {
+  @apply font-regular text-xl;
```

```suggestion
  @apply font-normal text-xl;
```
`font-normal`로 하셔야 `font-weight: 400` 이어요!